client/additional-methods-setup/wizard/task-item.scss

```diff
@@ -13,17 +13,18 @@
 		}
 	}
 
-	&__top-border {
+	&__top-border,
+	&__body {
 		min-height: 16px;
 		border-left: 1px solid #c3c4c7;
-		margin: 0 12px;
+		margin: 0 11px;
```

Comment:
Gone from 12px margin to 11px margin. While this might seem a weird dimension (considering that the icon/text has a width of 24px), we need to consider that the icon/text have a 1px border on left/right - so the calculation becomes `(24px (Width) - 2px (L/R borders)) / 2 = 11`